tests/Unit/Evolution/Systems/ForceFree/FiniteDifference/Test_Tags.cpp

```diff
@@ -0,0 +1,13 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include "Evolution/Systems/ForceFree/FiniteDifference/Tags.hpp"
+#include "Helpers/DataStructures/DataBox/TestHelpers.hpp"
+
+SPECTRE_TEST_CASE("Unit.Evolution.Systems.ForceFree.Fd.Tag",
+                  "[Unit][Evolution]") {
+  TestHelpers::db::test_simple_tag<ForceFree::fd::Tags::Reconstructor>(
```

Comment:
test the option tag using the appropriate function from `tests/Unit/Framework/TestCreation.hpp`

Reply:
Can we test it without any derived classes yet?

Reply:
I think you can make a derived class in your test in order to test it